more_itertools/tests/test_recipes.py

```diff
@@ -344,6 +344,50 @@ def test_combinatorics(self):
         )
 
 
+class UniquePowersetTests(TestCase):
+    """ Tests for ``unique_powerset``"""
+
+    def test_unique_powerset(self):
+        elements = [1, 1, 1, 2, 2, 3]
+        pss = set(mi.powerset(elements))
+
+        self.assertEqual(len(pss), 24)
```

Comment:
I think we can do without this check here - we're checking the length of `ups` below, and that `pss` and `ups` are the same.

Reply:
done in [9eba55](https://github.com/root-11/more-itertools/commit/9eba5565496d59305c3578a4e242ad069373ba1b)